### PR TITLE
Formatting and copy-editing for the Docker readme

### PR DESCRIPTION
**Pull request by tst-jcupp**

### CODE REVIEW

File: earth_enterprise/docker/README.md [hunk 1]
```diff
@@ -16,55 +16,53 @@ Add users that you want to have access to Docker to the `docker` group:
...
 
-    * Platform to build on,
-    * Source code origin: from the main repository, or a fork, from the
-      master, or another branch, or from the current repository clone on your
-      disk;
-    * Port to forward HTTP connections to;
-    * Custom image to run, e.g., containing extra data like tutorial
-      resources, or custom built globes.
+* Build platform
+* Source code origin: from the main repository, a fork, another branch,
```
Comment (tst-ppenev) [inline earth_enterprise/docker/README.md]: I think this one became less clear than the original. You could be cloning from `master` on the main repository, another branch on the main repository, `master` on a fork, another branch on a fork, or use current files on your disk.

File: earth_enterprise/docker/README.md [hunk 2]
```diff
@@ -16,55 +16,53 @@ Add users that you want to have access to Docker to the `docker` group:
...
+    or from the current repository clone on your disk
+* Port to forward HTTP connections
+* Custom image to run, e.g. one that contains extra data such as tutorial
+    resources or custom built globes
 
 These options are controlled by environment variables that are passed to the
-build scripts.  Since, it may become inconvenient to pass more than a couple
-of values on the command line, you could put the variable definitions in a
-shell script file instead, and source it before each command you execute.
+build scripts. Since it may become inconvenient to pass more than a couple
```
Comment (tst-ppenev) [inline earth_enterprise/docker/README.md]: I think you should use `a couple of` here.  We're not talking about exactly two values.

**Comment by tst-jcupp** [comment]
addressed the comments on wording

**Comment by tst-basilhuffman** [comment]
Other than issues related to running docker, looks good. No formatting issues.

**Pull request merged**